### PR TITLE
docs: refactoring the kind section to prepare for the additional docs

### PR DESCRIPTION
**Pull request by themr0c**
* Use Podman Desktop GUI rather than the command line.
* Reorganize to prepare for the additional documentation
* Based on features in v0.14.

references: https://github.com/containers/podman-desktop/issues/1955

### CODE REVIEW

File: website/docs/kubernetes/kind/configuring-podman-for-kind-on-windows.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   ```
+
+2. Set the Podman system default connection to connection that has the `-root` suffix:
+
+   ```shell-session
+   $ podman system connection default podman-machine-default-root
+   ```
+
+
+
```
Comment (themr0c) [inline website/docs/kubernetes/kind/configuring-podman-for-kind-on-windows.md:38]:
```suggestion
```

File: website/docs/kubernetes/kind/creating-a-kind-cluster.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+   * Provider type: `podman`
+   * HTTP port: `9090`
+   * HTTPS port: `9443`
+1. (Optionally) Click the **Show logs** button to display the logs.
+1. After successful creation, click on the **Go back to resources** button
+
+#### Verification
+
+1. In **Settings > Resources**, in the **Kind** tile, your Kind cluster is running.
+1. In the Podman Desktop tray, open the  **Kubernetes** menu, you can set the context to your Kind cluster.
```
Comment (jeffmaury) [inline website/docs/kubernetes/kind/creating-a-kind-cluster.md]: Would be good to indicate that the context is called **kind-cluster_name**

File: website/docs/kubernetes/kind/installing-kind.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+description: Kind is one way to get Kubernetes running on your workstation.
+keywords: [podman desktop, podman, containers, migrating, kubernetes, kind]
+tags: [migrating-to-kubernetes, kind]
+---
+
+# Installing the `kind` CLI
+
+#### Procedure
+
+* In the status bar, click on the **Kind** button and follow the prompts.
```
Comment (jeffmaury) [inline website/docs/kubernetes/kind/installing-kind.md]: Would be good to have an image of the toolbar there.

File: website/docs/kubernetes/kind/installing-kind.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+#### Verification
+
+1. The status bar doesn't display **Kind**.
+1. **Settings > Resources** contain a **Kind** tile.
```
Comment (jeffmaury) [inline website/docs/kubernetes/kind/installing-kind.md:19]: Same here having images would be a +

**Comment by themr0c** [comment]
@jeffmaury: I implemented your suggestions

**Review by jeffmaury — COMMENTED** [review]
Images are not of a very good quality is it me ?

Also https://github.com/containers/podman-desktop/pull/2008 will also start/stop/delete on Kind clusters once merged, FYI

**Comment by themr0c** [comment]
Images = screenshots from a Windows VM. It seems the screen is magnified and it adds some blurred effects. I am taking the screenshots again, but from Linux.

**Comment by themr0c** [comment]
Better screenshot pushed.

I took a screenshot from the left side of the tile only for better maintainability: changes in the right side don't require a docs update.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**